Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -607,7 +607,11 @@ async def kill_running():
 
             # kill the process (but asyncio is not notified immediately)
             proc.kill()
-            proc.wait()
+            try:
+                proc.wait()
+            except ProcessLookupError:
```

Comment:
Would you like to use https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises if it is always happened?

Reply:
It seems it’s not always raised, so I’ll revert to using a try–except block on [cb8d874](https://github.com/python/cpython/pull/137804/commits/cb8d8740e636fc38e9d27d74238b0726c2a37b11).